deploy/catalog/templates/apiserver.yaml

```diff
@@ -48,6 +70,14 @@ spec:
   selector:
     app: apiserver
   ports:
-  - protocol: TCP
+  {{ if .Values.insecure -}}
+  - name: insecure
+    protocol: TCP
+    nodePort: {{ default 30001 .Values.insecureServicePort }}
+    port: 80
+    targetPort: {{ default 8081 .Values.insecurePort }}
+  {{- end }}
+  - name: secure
+    protocol: TCP
     port: 6443
     targetPort: 6443
```

Comment:
@duglin are you okay with doing that in a follow-up?  this is just moving a chart around (it renders as a diff with how git counts changes).